app/src/pages/organization/organizationProjectsPage/projectsPageHeader/projectsPageHeader.jsx

```diff
@@ -94,6 +94,7 @@ export const ProjectsPageHeader = ({ hasPermission }) => {
 
 ProjectsPageHeader.propTypes = {
   hasPermission: PropTypes.bool,
+  showCreateProjectModal: PropTypes.func.isRequired,
```

Comment:
Let's rename to e.g. `onCreateProject` to not fully describe what this callback do within this component.
Let's leave the information that it is showing a modal in the parent component.